template/cordova/lib/log.js

```diff
@@ -47,10 +47,10 @@ var appTracingInitialState = null,
  * and prints them to console
  */
 module.exports.run = function(args) {
-    var startTime = new Date(new Date().getTime() - 10 * 60 * 1000).toISOString(), // show last 10 minutes by default
-        knownOpts = { 'minutes' : Number, 'dump' : Boolean, 'help' : Boolean },
-        shortHands = { 'mins' : ['--minutes'], 'h' : ['--help'] },
-        parsedOpts = nopt(knownOpts, shortHands, args, 0);
+    var startTime  = 10 * 60 * 1000; // show last 10 minutes by default
```

Comment:
Nitpick: It would be nice to rename startTime to something else. It isn't really a start time anymore.